### PR TITLE
Update mini magick

### PR DESCRIPTION
**Pull request by kareemshahin**
Forked over [herd](https://github.com/herdup/herd) repo and updated mini_magick due to high severity vuln.  Also updated ruby version to 2.3.7 for consistency with gravy

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,224 @@
+GIT
```
Comment (BrennanRoberts) [inline Gemfile.lock]: IIRC, gems should not maintain a lockfile.

Comment (kareemshahin) [reply Gemfile.lock]: good catch

File: herd.gemspec [hunk 2]
```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   # s.add_dependency 'pg'
   s.add_dependency 'active_model_serializers'
```
Comment (mdylanbell) [inline herd.gemspec]: Should we pin AMS to 0.8 since that's what we use in gravy?

**Comment by mdylanbell** [comment]
Do we need to consider maintaining `dev` and `master` branches?

File: herd.gemspec (hunk 2 above)
Comment (kareemshahin) [reply herd.gemspec]: probably wouldnt hurt

**Comment by kareemshahin** [comment]
dont know if we need to worry about maintaining this much as it should be replaced.  think just a `master` branch is fine

**Comment by leggomuhgreggo** [comment]
@kareemshahin Is this ready to rock? Trying to clean up outstanding PRs in advance of split restructure 🙏 
[Gravy PR](https://github.com/sweetgreen/gravy/pull/957)
[Sweetgreen PR](https://github.com/sweetgreen/sweetgreen/pull/3740)
[Sweetgreen DotCom PR](https://youtu.be/dQw4w9WgXcQ)

**Pull request closed without merging**